### PR TITLE
RSKIP 326 - btcDestinationAddress to Base58

### PR DESCRIPTION
**Pull request by kelvinator07**

### CODE REVIEW

**Comment by marcos-iov** [comment]
Perhaps you could include a bit more detail about how the event is currently used, the improvement proposal, and the motivation. See for example: 

- https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP181.md
- https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP185.md

File: IPs/RSKIP326.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+release_request_received(address indexed sender, string btcDestinationAddress, uint256 amount)
+```
+
+The btcDestinationAddress field will be in the base58 string format.
+
+#### Data
+
+- sender: the rsk address of the sender
+- btcDestinationAddress: the destination address of where the btc will be sent
+- amount: the value of btc sent
```
Comment (marcos-iov) [inline IPs/RSKIP326.md:43]: In satoshis? Btc? Maybe it's good to clear it out

File: IPs/RSKIP326.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+If `RSKIP326` is activated, the event will log the `btcDestinationAddress` field in the base58 string format.
+
+## Backwards Compatibility
+
+This change is a hard-fork and therefore all full nodes must be updated. SPV light-clients and Block explorers do not need to be updated.
```
Comment (marcos-iov) [inline IPs/RSKIP326.md:49]: If an explorer is decoding the events from the blockchain then it should be updated

File: IPs/RSKIP326.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+```
+release_request_received(address indexed sender, string btcDestinationAddress, uint256 amount)
+```
+
+The btcDestinationAddress field will be in the base58 string format.
+
+#### Data
+
+- sender: the rsk address of the sender
```
Comment (marcos-iov) [inline IPs/RSKIP326.md:41]: Maybe specify who the sender is, is it the user trying to do a pegout? Sending funds to the Bridge address?

File: IPs/RSKIP326.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+- btcDestinationAddress: the destination address of where the btc will be sent
+- amount: the value of btc sent
+
+If `RSKIP326` is activated, the event will log the `btcDestinationAddress` field in the base58 string format.
+
+## Backwards Compatibility
+
+This change is a hard-fork and therefore all full nodes must be updated. SPV light-clients and Block explorers do not need to be updated. 
+
```
Comment (marcos-iov) [inline IPs/RSKIP326.md:50]: You could also add a reference to the RSKIP where this event was added

File: IPs/RSKIP326.md (hunk 1 above)
Comment (kelvinator07) [reply IPs/RSKIP326.md]: Resolved

File: IPs/RSKIP326.md (hunk 2 above)
Comment (kelvinator07) [reply IPs/RSKIP326.md]: Resolved

File: IPs/RSKIP326.md (hunk 3 above)
Comment (kelvinator07) [reply IPs/RSKIP326.md]: Resolved

File: IPs/RSKIP326.md (hunk 4 above)
Comment (kelvinator07) [reply IPs/RSKIP326.md:54]: Resolved

File: IPs/RSKIP326.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+```
+release_request_received(address indexed sender, string btcDestinationAddress, uint256 amount)
+```
+
+The btcDestinationAddress field will be in the base58 string format.
+
+#### Data
+
+- sender: the RSK address of the release requester.
```
Comment (josedahlquist) [inline IPs/RSKIP326.md:41]:
```suggestion
- sender: the RSK address of the peg-out requester.
```

**Review by josedahlquist — APPROVED** [review]

**Review by jeremy-then — COMMENTED** [review]
LGTM.

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**